pkg/util/fileutil/file.go

```diff
@@ -51,6 +51,29 @@ func FileExists(filename string) (bool, error) {
 	return !info.IsDir(), nil
 }
 
+// FindFileAbsPath returns the absolute path for the given path and file
+func FindFileAbsPath(path, fileName string) (string, error) {
```

Comment:
Could we add a unit test for the func? Thanks!

Reply:
Sure

Reply:
Added. PTAL